src/components/CollapsibleTable.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <!-- Or I could just have used a datalist -->
+  <details open>
+    <summary>
+      <slot name="title"></slot>
```

Comment:
Oh nice, slots are that easy to use? Looks like I'm learning something new 👍

Reply:
https://vuejs.org/v2/guide/components-slots.html#Named-Slots